storage/kv_test.go

```diff
@@ -736,6 +737,14 @@ func TestKVSnapshot(t *testing.T) {
 	}
 }
 
+func waitTimeoutError(t *testing.T, msg string) {
```

Comment:
let's move this to pkg/testutil. Also we should call this fatalStack. This has nothing to do with `wait` or `timeout` or `error`. The function name should describe what it does rather than whose is using it currently.